pkg/configuration/config.go

```diff
@@ -234,6 +234,18 @@ func LocalConfig(cmd *cobra.Command) models.ScopedOptions {
 		}
 	}
 
+	flagSet = cmd.Flags().Changed("config-dir")
+	if flagSet || localConfig.UserConfigDir.Value == "" {
+		localConfig.UserConfigDir.Value = cmd.Flag("config-dir").Value.String()
+		localConfig.UserConfigDir.Scope = "/"
+
+		if flagSet {
+			localConfig.UserConfigDir.Source = models.FlagSource.String()
+		} else {
+			localConfig.UserConfigDir.Source = models.DefaultValueSource.String()
+		}
+	}
```

Comment:
Does it make sense to include the config dir in the configuration output? We don't do this for other values like `config-file`, as we only print values that are stored in the actual config file.

Reply:
I had just gone through trying to reproduce the stuff that existed for other settings that were pulled from env-vars. As long as this keeps showing up in `doppler configure debug` (which it does) after removing this, seems fine to me! I agree that it shouldn't be stored in a config file somewhere (wasn't actually clear that this is what was being done here).

Reply:
I tried backing out the changes in `models/config.go`, but then it stopped showing up in `doppler configure debug` output. That does seem like the sort of thing we'd want to show up there. Maybe there's some subset of what's being done in `models/config.go` that it can be paired down to?

Reply:
The table output by `doppler configure debug` is exclusively for values that can be stored in the config file. Above the table, we also print the path of the config file that's used. You'll want to add the config dir there, which won't require modifying `models/config.go`.

Reply:
Ah, I see. I'm following now. Pushed these changes up!